tools/pv_tree_generator/util.py

```diff
@@ -51,4 +141,52 @@ def _read_pop_obs_spec():
         PopObsSpec(pos.pop_type, pos.mprop, pos.stat_type, list(pos.cprop),
                    cpv, pos.name))
   return result
+  
+def _read_stat_var():
+    """Read all the statistical variables"""
+    #read the dcid of all statistical variables
+
+    #currently we read it from the downloaded .csv file
+    #in the future, we will change to query it from bigquery,
+    #or from calling API of datacommons
+    sv_dcid = open("statistical_variables_dcid.txt",'r').read().splitlines()
```

Comment:
Please check the offline discussion on how to read the stats var dcid using SPARQL

Reply:
done